packages/sources/alpine/src/endpoint/tvl.ts

```diff
@@ -0,0 +1,35 @@
+import { Validator, util } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { ethers } from 'ethers'
+import vaultAbi from '../abi/vault.json'
+
+/**
+ * This endpoint gets us the TVL of a vault on ethereum.
+ */
+export const supportedEndpoints = ['tvl']
+
+export const inputParameters: InputParameters = {
+  vaultAddress: true,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const { vaultAddress } = validator.validated.data
+  const network = validator.validated.data.network || config.network
+
+  const rpcUrl = network == 'ETH' ? util.getEnv('ETHEREUM_RPC_URL') : util.getEnv('POLYGON_RPC_URL')
```

Comment:
This checks `network == 'ETH'` while other places use `"ETHEREUM"`. This needs to be changed, but you could also define the names as consts to make sure it stays consistent through the entire adapter.

Same suggestion as above applies here (read RPC URLs from config instead of reading env vars here)